scrapper.py

```diff
@@ -29,6 +29,9 @@ def __init__(self, seed_urls, max_articles: int):
         pass
 
     def _extract_url(self, article_bs):
+        response = requests.get('https://periodical.pstgu.ru/ru/series/index/3')
```

Comment:
this is an incorrect place for downloading the page. this function is responsible for extraction of standalone artile URL